filebeat/docs/configuring-howto.asciidoc

```diff
@@ -35,6 +35,7 @@ The following topics describe how to configure Filebeat:
 * <<configuring-ingest-node>>
 * <<{beatname_lc}-geoip>>
 * <<configuration-path>>
+* <<input-file-log-rotation>>
```

Comment:
I don't think this is the right location for this information. Do you think it would make sense to have this topic in the troubleshooting section of the docs and link to it from the section about the log input? You could change the title to be "Log rotation results in lost log messages or duplicated events."  I just opened a PR to change the format of the troubleshooting section so we can add individual topics instead of having everything in a single FAQ-style topic: https://github.com/elastic/beats/pull/12992. 

If you think that it's important to have this information in the section under configuration, then it should appear under "Configure inputs".

Reply:
I am OK with moving it to the troubleshooting section.